### ISSUE TITLE
What do we know about blocked numbers?

### ISSUE THREAD

**Issue by Pietertje007** [venomous0x/WhatsAPI#358]
What do we know about when a number is blocked by whatsapp?

I just had a number that was blocked within 2h30m and I only send/received 38 messages in total. So it's not the message count...

On 2 other cards, I thought it happened after trying to send html tags to my phone. But that does not seem correctly. Last one blocked, no html tag was sent. Also other cards have not been blocked and  did sent tags with them.

An other interesting thing is that blocking seems te happen when you login. My last blocked number, I succesfully sent a message and when I logged in again, 2 minutes later, it was blocked. 

Amount of logins is also not the case. I have a number that I use a lott and it is never blocked. The last one blocked, I logged in, just about 4 times.

So basically it seems totally random for me....

Anyone here to add some ideas about when numbers are blocked?

**Comment by AyushBhat** [comment]
same is happening with me and now I am so much annoyed with it
